composer.json

```diff
@@ -18,7 +18,10 @@
     "myclabs/php-enum": "^1.6"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5"
+    "phpunit/phpunit": "^5",
+    "guzzlehttp/guzzle": "^6.3",
+    "php-http/guzzle6-adapter": "^2.0",
+    "http-interop/http-factory-guzzle": "^1.0"
```

Comment:
remove those dependencies please
to allow the CI to run well, add those dependencies in the travis.yml instead :)

Reply:
I'm checking if there is no way to avoid those